src/assets/contributors-map.geojson

```diff
@@ -45,6 +45,21 @@
           36.037803
         ]
       }
-    }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "firstName": "Scott",
+        "githubId": "scooterh4",
+        "favoriteCrag": "4cb01c90-8df7-5158-be07-ba1c83ee3c0c"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          51.047,
+          -115.1566
```

Comment:
I think the coordinates need to be flip, ie `[-115.1566,  51.047]`

You can paste the entire file in  https://geojson.io/#map=11.01/51.0278/-115.1289 to verify the geojson format.
If the format is correct you will see a globe with drop pins.